### DISCUSSION TITLE
Multiple LDAP URLs

### DISCUSSION THREAD

**Discussion by v-bulynkin** [requarks/wiki#6520] [Help / Questions] [answered]
Is it possible to use several LDAP URLs in case if the first server is unavailable?

**Comment by P4mes4n** [comment, marked as answer]
You should be able to just use the Domain Name as target.
If your DNS is set up correctly it should just find an available DC on its own.

**Reply by v-bulynkin** [reply]
Yes, it works with LDAP URL like `ldap://example.com:389`.
Thanks!
